subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseProjectFixture.groovy

```diff
@@ -72,7 +72,7 @@ class EclipseProjectFixture extends IdeProjectFixture {
     }
 
     void assertHasNatures(String... natures) {
-        assert this.project.natures.nature*.text() == natures as List
+        assert this.project.natures.nature*.text() as Set == natures as Set
```

Comment:
❓ Why do we no longer care to check the ordering of these, and the buildCommands below?

Reply:
I figured the ordering was not significant. Though, in case it is, I've reversed this change.